src/model/AbstractNodeVersion.ts

```diff
@@ -0,0 +1,29 @@
+import { debug } from '../utils';
+
+export abstract class AbstractNodeVersion {
+    public major: string | undefined;
+    public minor: string | undefined;
+    public patch: string | undefined;
+
+    public toString(): string | undefined {
+        if (this.major && this.minor && this.patch) {
+            return [this.major, this.minor, this.patch].join('.');
+        } else {
+            debug('Can not get Node version!');
```

Comment:
Shouldn't this be more of a warning or an error maybe? 🤔

Reply:
Yeah, we could make a warning 👍